animation_nodes/nodes/gpencil/gp_stroke_info.py

```diff
@@ -21,5 +21,9 @@ def create(self):
         self.newOutput("Text", "Display Mode", "displayMode", hide = True)
 
     def execute(self, stroke):
-        return (stroke.vertices, stroke.strengths, stroke.pressures, stroke.uvRotations, stroke.lineWidth,
-                stroke.drawCyclic, stroke.startCapMode, stroke.endCapMode, stroke.materialIndex, stroke.displayMode)
+        strengths = DoubleList.fromValues(stroke.strengths.copy())
+        pressures = DoubleList.fromValues(stroke.pressures.copy())
+        uvRotations = DoubleList.fromValues(stroke.uvRotations.copy())
```

Comment:
You shouldn't copy here. Copying is the responsibility of whoever edits the data.

Reply:
Removed. Thanks for the info.